posthog/models/instance_setting.py

```diff
@@ -18,12 +23,30 @@ class Meta:
     def value(self):
         return json.loads(self.raw_value)
 
+    def delete(self, *args, **kwargs):
+        cache.delete(self.key.replace(CONSTANCE_DATABASE_PREFIX, ""))
```

Comment:
is there a reason to not use this prefix in the cache as well?

Prevents clashes with other random things in the cache with the same name

Reply:
Tbh I don't know why the prefix exists. Seems to me we should have gotten rid of it together with `django-constance`.